src/io/tree.cpp

```diff
@@ -204,6 +204,14 @@ void Tree::AddPredictionToScore(const Dataset* data,
   }
 }
 
+void Tree::AddMaxValueToScore(double& score) const {
```

Comment:
why not just return the max/min values, and add outside?

Reply:
I can do it, it just felt a little more coherent with methods in the same class that are accumulating score values, I know that in those cases they pass pointers because they are arrays. Just let me know which solution you like better.

Reply:
I think just returning the max/min score is better, as it is more standalone and will be possible used by others.

Reply:
Done, thank you